launch/launch/actions/execute_process.py

```diff
@@ -29,6 +29,7 @@
 from typing import List
 from typing import Optional
 from typing import Text
+from typing import Tuple  # noqa: F401
```

Comment:
Why is F401 needed here, looks like you are using Tuple?

Reply:
Because we have to support python 3.5, we cannot use the type annotation for member variables syntax which was added in 3.6. So we use the type comment, which is the only place I use Tuple. So flake8 doesn’t count that as using it so I suppressed the unused import warning.

Reply:
Because we have to support python 3.5, we cannot use the type annotation for member variables syntax which was added in 3.6. So we use the type comment, which is the only place I use Tuple. So flake8 doesn’t count that as using it so I suppressed the unused import warning.